api/HTMLFrameElement.json

```diff
@@ -0,0 +1,532 @@
+{
+  "api": {
+    "HTMLFrameElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLFrameElement",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": false,
```

Comment:
I still feel like these should be "standard_track": true -- this is a standardized, but deprecated, interface, which is very different from a deprecated interface that was never standard. I'm not sure we ever actually completed the debate on he matter

Reply:
I agree, the behavior was/is standardized, but it's now deprecated, so standard_track can be true here.